components/alert/no-js-alerts/no-js-alert.css

```diff
@@ -30,6 +30,7 @@
 .alert{
     border: 1px solid var(--light-gray);
     color: var(--light-gray);
+    justify-content: flex-start;
     margin: 1rem 0rem;
     padding: 1rem;
     width: 80%;
```

Comment:
You can use max-width so that it doesn't shrinks on smaller screen

Reply:
okay will do these changes, thankyou so much for reviewing!!